deletelistView.php

```diff
@@ -0,0 +1,123 @@
+                boardID
+            ) VALUES ( 
+                :listTitle, 
+                :boardID
+            ) 
+        "; 
+        $query_params = array( 
+            ':listTitle' => $_POST['name'], //Insert the user's input into the database
+            ':boardID' => 1 //Placeholder until we develop the board view function
+        ); 
+        $stmt = $db->prepare($query); 
+       	$result = $stmt->execute($query_params); 
+}
+if (!empty($_POST['name'])){
+	//If the user submitted a new delete list request we run this code
+	$query = " 
+            delete FROM taskList 
+            WHERE listTitle = ':listTitle' "; 
+        $query_params = ':listTitle' => $_POST['name']; //Insert the user's input into the database 
+        $stmt = $db->prepare($query); 
+       	$result = $stmt->execute($query_params); 
+}
+
+// get complete details for a list
+$stmt = $db->prepare('SELECT * FROM taskList ORDER BY `listID` ASC');
+            $stmt->execute();       
+            $lists = $stmt->fetchAll();
+echo "<div class='row'>";
+foreach ($lists as $list){
+	//This code is run FOR EACH list that exists.
+	echo "
+	<div class='col-xs-3'>
+	<div class='card'>
+	<div class='card-header'>".$list['listTitle']."</div>
+	<ul class='list-group list-group-flush'>
+	";
+  
+    
+	$stmt = $db->prepare('SELECT * FROM task WHERE `tl.listID` = ?');
+            $stmt->execute(array($list['listID']));       
+            $tasks = $stmt->fetchAll();
+            foreach ($tasks as $task){
+            	//This code is run FOR EACH task in each list. Here we output an html row with some data.
+            	echo "<li class='list-group-item'>".$task['taskTitle'].": ".$task['description']."</li>";
+            }
+    echo " 
+    </ul>
+	</div>
+	</div>";
+}
+echo "</div>";
+//Below we have the code for our "modal" which pops up when the user clicks the add new list button. It is a simple form that submits back here for a "page refresh" with the new list.
+?>
+<div class="modal fade" id="createList" tabindex="-1" aria-labelledby="createList" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">Add New List</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <form action="" class="form-newList" method="POST">
+
+            <label for="inputName" class="sr-only">List Name</label>
+            <input type="text" name="name" value="" class="form-control" placeholder="List Name" required/>
+            <button class="btn btn-lg btn-primary btn-block" type="submit" value="Register">Create</button>
+        </form>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="modal fade" id="createList" tabindex="-1" aria-labelledby="createList" aria-hidden="true">

```

Comment:
- This block (line 100 - 119) can also be moved to `listView.php`

- Change the form input name to the `$_POST` variable name you changed above